routes/api.php

```diff
@@ -7,11 +7,11 @@
 
 
 Route::middleware('auth')->group(function () {
-    Route::get('/employee', [EmployeeController::class , 'index'])->middleware('auth:sanctum')->name('employee');
+    Route::get('/', [EmployeeController::class , 'index'])->middleware('auth:sanctum')->name('employee');
```

Comment:
please setup a group middleware instead of individual @axnish27

Reply:
okay sure @Raza9798